packages/utilities/fast-web-utilities/src/numbers.ts

```diff
@@ -17,6 +25,13 @@ export function wrapInBounds(min: number, max: number, value: number): number {
  * Ensures that a value is between a min and max value. If value is lower than min, min will be returned.
  * If value is greater than max, max will be returned.
  */
-export function limit(min: number, max: number, value: number): number {
+export function limit(
+    min: number | null,
```

Comment:
It seems that these changes and the changes in `query.ts` are only necessary because we have tests that invoke with invalid arguments.

I think we should probably just remove those tests and rely on the compiler to ensure arguments are accurate. If an author provides invalid arguments that isn't the responsibility of these functions to guard against.

Reply:
What I mean is that we shouldn't change these function signatures. Can you revert them and adjust the tests as necessary so that tests compile? Thanks!